include/constants/moves.h

```diff
@@ -939,6 +939,13 @@
 
 // there are a bunch more here but that is fine
 
+#define NUM_OF_CANONICAL_MOVES 923
+
+// make.py doesn't support constants for now
```

Comment:
what needs move constants in make.py?

Reply:
If I write it like this:
```c
#define NUM_OF_CUSTOM_MOVES 0
#define NUM_OF_MOVES (NUM_OF_CANONICAL_MOVES+NUM_OF_CUSTOM_MOVES)
```
The `make.py` complains it can't convert `(NUM_OF_CANONICAL_MOVES+NUM_OF_CUSTOM_MOVES)` into a number